docs/tizen-distribution/validation/guidelines/age-rating-guide.md

```diff
@@ -0,0 +1,37 @@
+# Age Rating Guide
+
+1. Age Rating Categories
+
+   Age Rating is used to ensure that applications available in the Tizen Store are labeled by age according to the content. Age rating provides guidance to users, particularly parents, to help them decide whether to download a particular application or not. The rating of an application confirms that it is suitable for users over a certain age. Age rating is classified in the following way:
+
+   - All : Applications that everyone can play.
+   - 12+ : Applications that child under the age of 12 cannot play unless accompanied by adults.
+   - 16+ : Applications that children and teenagers under the age of 16 cannot play unless accompanied by adults.
+   - 18+ : Applications that teenager and children (under 18) cannot play.
+
+   Content rating in the Tizen Store has been created according to international age rating standards for video/game content such as ESRB, PEGI. Therefore, if any content has an international age rating level, the level will be accepted without further review.
+
+   **Table: Age rating**
+
+   | Tizen | ESRB   | PEGI | GRB  | MJ/DEJUS |
+   | ----- | ------ | ---- | ---- | -------- |
+   | ALL   | EC     | 3+   | All  | ER/L     |
+   | ALL   | E      | 7+   | All  | 10+      |
+   | 12+   | E10+   | 12+  | 12+  | 12+,14+  |
+   | 16+   | T      | 16+  | 15+  | 16+      |
+   | 18+   | Mature | 18+  | 18+  | 18+      |
+
+   ​
```

Comment:
There is useless special character (<200b>) on this line.

Pleae remove <200b> on other files.